### PR TITLE
Port OpenCV code to Java OpenCV bindings

### PR DESCRIPTION
**Pull request by Erackron**
Also created custom NamedWindow and ImagePanel classes to imitate the NamedWindow class from the C++ OpenCV api

Tests for the ImageProcessing class are rather difficult,
but tests for the BlobDetectionFrameHandler might still be feasible.
The NamedWindow and ImagePanel classes have 77% line coverage

### CODE REVIEW

**Comment by MitchellHop** [comment]
all looks fine

**Comment by martinck** [comment]
Only not able to merge pull request yet(?) Code itself seems allright

File: image-processing/src/main/java/nl/tudelft/contextproject/imageprocessing/framehandlers/BlobDetectionFrameHandler.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        if (!capture.read(next)) {
+            throw new Exception("Unable to open camera");
+        }
+
+        process(previous, next);
+
+        end = System.currentTimeMillis();
+        System.out.printf("duration: %.2fs\n", (end - start) / 1000f);
+        previous.release();
+        previous = next.clone();
```
Comment (LiamClark) [inline image-processing/src/main/java/nl/tudelft/contextproject/imageprocessing/framehandlers/BlobDetectionFrameHandler.java:68]: this clone call can be avoided

**Comment by LiamClark** [comment]
looks fine

**Pull request merged**